target/linux/ath79/dts/tp9343_tplink_tl-wr941hp-v1.dts

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	compatible = "tplink,tl-wr941hp-v1", "qca,tp9343";
+	model = "TP-Link TL-WR941HP v1";
+
+	aliases {
+		label-mac-device = &wmac;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+        ranext {
+            label = "blue:ranext";
+            gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+        };
```

Comment:
Broken indent. What's that anyway?

Reply:
It's a "range extender" button in stock firmware, I don't know exactly what it does. I change the label to "re" like other dts files.

Reply:
Ah, thanks, finally I know what this cryptic RE stands for ...